### ISSUE TITLE
How to train with full supervised mode?

### ISSUE THREAD

**Issue by wqhIris** [Jia-Research-Lab/Context-Aware-Consistency#3]
Thank you for your nice work!
I was a little wondering about :
1. the setting of training the model with total labeled 10,582 samples;
2. when employing "Full labeled data", the differences between settings of ”SupOnly“ and "CAC".

**Comment by X-Lai** [comment]
Thanks for your question. When using the 'full labeled data', the 'SupOnly' setting means only using the cross entropy loss from the predictions and the ground-truth, while the 'CAC' setting also adds our proposed DC Loss. As for the details, because there are no unlabeled images in the fully supervised setting, so we use all images to compute the DC Loss.

**Comment by wqhIris** [comment]
Thank you a lot, I got it.
